modules_basic/public.js

```diff
@@ -34,7 +34,7 @@ exports.create = function (api) {
 
         pull(
           u.next(api.sbot_log, {old: false, limit: 100}),
-          Scroller(div, content, api.message_render, true, false)
+          ScrollNotify(div, Scroller(div, content, api.message_render, true, false))
```

Comment:
In my code I changed `div` to be called `container`. I think the div you're giving to `ScrollNotify` should  not be the container, rather the hypertab tab, which are defined in `modules_core/tabs` right ?